### DISCUSSION TITLE
Will this tool support the upcoming Ryzen 7040HS Phoenix CPUs?

### DISCUSSION THREAD

**Discussion by SandyBridgeHX** [DavidS95/Smokeless_UMAF#20] [Q&A] [answered]
First of all, thanks David for hard and wonderful work! I successfully enabled the S3 sleep mode instead of Modern Stanby S0 on my Yoga Pro Laptop (6800HS edition) without any side effects. Thanks for this tool, I won't have to worry about the heat and power consumption of my laptop when it's on standby with the lid closed, and the system will power up my hard drive a lot less.

As the AMD new CPU 7040HS will luanch on upcoming laptop SKUs, I probably plan to switch my laptop to the new one. Due to Microsoft limitations, new laptop might be strict to S0 sleep mode like the one I'm using. Thus, I think the UniversalAMDFormBrowser tool will still be the only way to enable the S3 sleep mode.  If I may be so bold to ask, will this tool support the future 7040HS?

Thanks a lot!!

**Comment by DavidS95** [comment, marked as answer]
Test the newest BETA uploaded it should better support Phoenix APUS

**Reply by SandyBridgeHX** [reply]
Thank you David! Glad to hear this great news ^_^
